end_to_end/signal_handler_test.go

```diff
@@ -11,6 +13,14 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func CreateOutput(cmd *exec.Cmd) (*bufio.Scanner, strings.Builder) {
+	outPile, _ := cmd.StdoutPipe()
+	outScanner := bufio.NewScanner(outPile)
+	var output strings.Builder
+	cmd.Stderr = &output
```

Comment:
we use output variable for stderr? May be call it 'stderr'?

Reply:
We use this variable for both outputs.

Reply:
where can we see assign of output variable to stdout?

Reply:
This happens when creating a Pipe.

Reply:
Sorry, still can't understand it. It returns variable _outScanner_ which built on top of outPipe which is stdoutPipe. And we return string builder variable _output_ which connected to stderr. 
How it comes that variable _output_ connected to stdout?
And wouldn't it be better to return combined stdout/stderr here with only one return variable?

Reply:
We create a pipe attached to stdout. Then we create a scanner using this pipe. Then in getOutput we read from the scanner and write to the output buffer.

Reply:
The I'll ask again - why variable attached to stderr called 'output'? May be 'stderr' would be better name?
And second question - can we combine stdout and stderr in CreateOutput function, call it CreateCombinedOutput() and analyze combined stread? The code will be simpler.
Like this: https://stackoverflow.com/questions/35994907/go-combining-cmd-stdoutpipe-and-cmd-stderrpipe

Reply:
> The I'll ask again - why variable attached to stderr called 'output'? May be 'stderr' would be better name?

Because not only stderr is written there, but also stdout is written through reading from the scanner.

> And second question - can we combine stdout and stderr in CreateOutput function, call it CreateCombinedOutput() and analyze combined stread? The code will be simpler.
Like this: https://stackoverflow.com/questions/35994907/go-combining-cmd-stdoutpipe-and-cmd-stderrpipe

Good idea.